api/v1beta1/spiaccesstoken_types.go

```diff
@@ -182,13 +182,13 @@ type SPIAccessTokenList struct {
 
 // EnsureLabels makes sure that the object has labels set according to its spec. The labels are used for faster lookup during
 // token matching with bindings. Returns `true` if the labels were changed, `false` otherwise.
-func (t *SPIAccessToken) EnsureLabels(detectedType ServiceProviderType) (changed bool) {
+func (t *SPIAccessToken) EnsureLabels(detectedType string) (changed bool) {
```

Comment:
Because I've changed `GetType()` from `api.ServiceProviderType` to `config.ServiceProviderType` https://github.com/redhat-appstudio/service-provider-integration-operator/pull/400/files#diff-ffcc30208e30f14119a24805225677b1fb551e0ee1396df0857cedbd7c9789fdR55
We can't use `config` package here and I didn't want to do any conversions. I'll take a look again.

Reply:
I've moved the function to spiaccesstoken_controller.go